pkg/stanza/operator/input/tcp/tcp.go

```diff
@@ -59,8 +59,8 @@ func NewConfig(operatorID string) *Config {
 
 // Config is the configuration of a tcp input operator.
 type Config struct {
-	helper.InputConfig `yaml:",inline"`
-	BaseConfig         `yaml:",inline"`
+	helper.InputConfig `mapstructure:",squash" yaml:",inline"`
```

Comment:
Do you want to maintain yaml and json tags as well?

Reply:
For now, but I would like to remove them as a separate effort. There will be a bunch of related code that can be pulled out too, so I think a separate PR will make sense.

Reply:
sure, just checking the long term commitment that we make here.